css/design_system/utils/text.scss

```diff
@@ -0,0 +1,82 @@
+@mixin docstring($class, $prop) {
+  /**
+   * @title #{$class}
+   * @category utils
+   * @util_type #{$prop}
+   *
+   * @example
+   * <p class="#{$class}">Example text</p>
+   */
+}
+
+@mixin docstring_messy($class, $prop) {
+  /**
+   * @title #{$class}
+   * @category utils
+   * @util_type #{$prop}
+   *
+   * @example
+   * <p class="#{$class}">exaMple tEXT</p>
+   */
+}
```

Comment:
Can we parameterize the example text and just have a single `docstring` mixin?

Reply:
Yes!! Thank you.